### DISCUSSION TITLE
X4: Foundations now uses Jolt!

### DISCUSSION THREAD

**Discussion by DocAce** [jrouwe/JoltPhysics#413] [Show and tell]
With the recently released 6.0 patch (in Beta for now) X4: Foundations, the space simulation I'm working on, has successfully switched from Bullet to Jolt 🥳

**Comment by jankrassnigg** [comment]
Congratulations! A looong time ago (I guess before you were there) I was briefly interning at Egosoft. Pretty cool to see how it became better and better over the years!

**Reply by DocAce** [reply]
Cool! One of our oldest colleagues also doesn't know you, so that must have been a _really_ long time :)

**Reply by jankrassnigg** [reply]
2005 I think. And only for 3 weeks or so. All I remember is the scripting language... "PC" or whatever its name was 😅

**Reply by DocAce** [reply]
Oh, that's not as long as I thought, I started in 2007 :)
And that language was KC (fun fact: it's short for Ketchup Code, it was initially developed for an advert game called Ketchup Connection!)
We're not using that anymore, but we still have our own internally developed custom scripting languages :D

**Comment by jrouwe** [comment]
Super cool! I just gifted the game to my son who promised to Beta test it (he has much more time on his hands so he can build a nice space station for me to walk around in and try things out) :)

I'm surprised at the speed with which it came out, only a couple of days ago you were having troubles walking around a space ship. Did you solve that in the end? (must have been quite a crunch)

**Reply by DocAce** [reply]
It's true, this public beta is coming in much hotter than usual, but as it's opt-in we're okay with that as long as there are no known game stoppers :) Our fanbase also skews a bit on the older and more technically inclined side, so they're pretty accepting of a really rough start to a beta phase.

The walking around part has indeed been fixed completely. I haven't yet gotten around to merging in your latest change in that regard, though, so I'm still using my local workaround.
